oat-sa/extension-lti-outcomeui
# highlight eraser

Ticket: https://oat-sa.atlassian.net/browse/MS-469

### What's changed?

- build script is added
- Highlight tray markup is improved to adapt the future changes
- eraser function is added

### How to test?

- Check out the branch and open a task
- Make sure `tao-core-ui-fe` library is installed with this version: https://github.com/oat-sa/extension-lti-outcomeui/pull/18
- Click on the highlighter icon and save some highlights
- Now, click on the eraser and erase those highlights by clicking on the text
- refresh to make sure changes are saved as well

### Related PRs

https://github.com/oat-sa/tao-core-ui-fe/pull/196/files
